src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -62,5 +152,34 @@ private static UIElement CreateElement(string text, double lineHeight, TextForma
             border.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
             return border;
         }
+
+        /// <summary>
+        /// Determines if the border is being moused over and shows the info accordingly
+        /// </summary>
+        private void Border_ToolTipOpening(object sender, ToolTipEventArgs e)
+        {
+            var border = (Border)sender;
+            e.Handled = true;
+
+            bool KeepOpen()
+            {
+                var mousePoint = Mouse.GetPosition(border);
+                return !(mousePoint.X > border.ActualWidth || mousePoint.X < 0 || mousePoint.Y > border.ActualHeight || mousePoint.Y < 0);
+            }
+
+            var toolTipPresenter = _toolTipService.CreatePresenter(_textView, new ToolTipParameters(trackMouse: true, ignoreBufferChange: false, KeepOpen));
+            var startToolTip = StartToolTipServiceAsync(toolTipPresenter);
```

Comment:
Let's explicitly assign this like:

```suggestion
            _ = StartToolTipServiceAsync(toolTipPresenter);
```

So it's clear we're not processing that task further.

Reply:
Makes sense, fixed